src/java.management/share/classes/javax/management/openmbean/ArrayType.java

```diff
@@ -92,6 +92,8 @@
  * ArrayType<String[][][]> a19 = new ArrayType<String[][][]>(1, a18);
  * }</pre>
  *
+ * @param <T> the Java type that instances described by this type must
+ * have.
```

Comment:
Instead of "instances" - would it be more correct to say "array elements"?

Reply:
Will change to "the Java component type that arrays described by ArrayType must have"